keyboards/ymdk/ymd62/halconf.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2020 QMK
+ * Copyright 2024 Richard Dawe (@richdawe)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * This file was auto-generated by:
+ *    `qmk chibios-confmigrate -i keyboards/handwired/onekey/bluepill/halconf.h -r platforms/chibios/common/configs/halconf.h`
+ */
```

Comment:
I used `keyboards/handwired/onekey/bluepill_uf2boot` as the basis for my port, because that is also a board using an STM32F103 that supports uf2boot, like the YMD62.

The `INFO_UF2.TXT` for the YMD62 contains:

```
UF2 Bootloader v2.1.1-dirty
Model: MT KB 
Board-ID: STM32F103-blue-pill-v0
```

It's using a Geehy APM32F103CBT6 microcontroller.